mtlpy/views.py

```diff
@@ -88,3 +93,8 @@ def sponsorship(request):
                 .order_by('ordering'))
     return render_to_response('sponsorship.html', {"sponsors": sponsors},
                               context_instance=RequestContext(request))
+
+
+def debug(request):
+    log.error("Debug request env=%s headers=%s", os.environ, request.META)
+    return JsonResponse({})
```

Comment:
I was returning some payload, not needed anymore. Will be handy to debug further.